server/game/cards/01_SOR/Tarkintown.ts

```diff
@@ -16,8 +16,7 @@ export default class Tarkintown extends BaseCard {
         this.setEpicActionAbility({
             title: 'Deal 3 damage to a damaged non-leader unit',
             targetResolver: {
-                cardTypeFilter: WildcardCardType.NonLeaderUnit,
-                cardCondition: (card) => (card as UnitCard).damage !== 0,
+                cardCondition: (card) => card.isUnit() && card.damage !== 0,
```

Comment:
Does the isUnit() method only return true for non-leader units?

Reply:
The word `Unit` is used in the code the same way it is in the game rules, so it can refer to leader units, token units, and units that are neither of the above

Reply:
Then wouldn't this allow Tarkintown to target a leader unit?

Reply:
Good question. That isn’t an issue because GameSystems can declare a set of card types that are legally allowed to target, and the DamageSystem can’t target undeployed leaders

Reply:
Ope you’re right that’s a bug 😅